ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -29,29 +41,46 @@ import WatchedImagesModal from '../WatchedImages/WatchedImagesModal';
 import UnwatchImageModal from '../WatchedImages/UnwatchImageModal';
 import VulnerabilityStateTabs from '../components/VulnerabilityStateTabs';
 import useVulnerabilityState from '../hooks/useVulnerabilityState';
+import DefaultFilterModal from '../components/DefaultFilterModal';
 
-const emptyStorage: VulnMgmtLocalStorage = {
-    preferences: {
-        defaultFilters: {
-            // TODO: re-add default filters to include critical, important, and fixable
-            Severity: [],
-            Fixable: [],
-        },
-    },
-};
```

Comment:
This was moved inside the component render since we need the default filters for new users to be based on the value of a feature flag.